functions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**A function that connects to the SQL database and retrieves an object.
+ * @return object
+ */
+function getDb(): object {
+    return new PDO ('mysql:host=db; dbname=sami-collection','root', 'password');
+}
+
+/**This function takes a database object and selects the items from the database to output an array.
+ * @param object $db is the object from the getDb function that is a connection to the database content.
```

Comment:
please add a line break between the description param and return lines

Reply:
(please do elsewhere in the file too)

Reply:
Have done this